source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -454,7 +454,8 @@ export default class WalletSendForm extends Component<Props, State> {
         this.isLatestTransactionFeeRequest(
           this.state.feeCalculationRequestQue,
           prevFeeCalculationRequestQue
-        )
+        ) &&
+        !this.selectedAssetsAmounts.includes('0')
```

Comment:
@yakovkaravelov I'm not sure that this is correct. What will be if the asset amount is `10` ? Maybe you should check if the asset added and not empty.

Reply:
@tomislavhoracek Yeah, you are right. This conditional check was just copied from different place of this same file...
Need to fix this one and that one as well.

Reply:
@tomislavhoracek It is not a string, it is an array of string.
So `10` can't satisfy above condition.

Reply:
@yakovkaravelov oke, then we are good!